File: 프로그래머스/2/42746. 가장 큰 수/가장 큰 수.js
function solution(numbers) {
    const strs = numbers.map(num => num.toString());
    
    strs.sort((a, b) => (b + a) - (a + b));
    
    const answer = strs.join("");
    
    return answer[0] === "0" ? "0" : answer;
}

File: 프로그래머스/2/42746. 가장 큰 수/가장 큰 수.py
def solution(numbers):
    strs = list(map(str, numbers))
    # '3333333333' > '30303030303030303030'
    strs.sort(key=lambda x:x*10, reverse=True)
    return ''.join(strs) if int(strs[0]) != 0 else '0'